frontend/src/app/projects/dashboard/layout.tsx

```
@@ -0,0 +1,17 @@
+import React from 'react'
+import ProjectsDashboardNavBar from 'components/ProjectsDashboardNavBar'
+
+const ProjectsHealthDashboardLayout: React.FC<{ readonly children: React.ReactNode }> = ({
+  children,
+}) => {
+  return (
+    <div className="flex flex-col md:flex-row">
+      <ProjectsDashboardNavBar />
+      <main className="flex-1 p-4" role="main">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<main>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)